erpnext/stock/doctype/quality_inspection/quality_inspection.js

```diff
@@ -63,12 +63,23 @@ frappe.ui.form.on("Quality Inspection", {
 frappe.ui.form.on("Quality Inspection Reading", {
 	status: function (frm, cdt, cdn) {
 		let row = locals[cdt][cdn];
+		let not_rejected = true;
```

Comment:
Couldn't this be:

```suggestion
		let rejected = false;
```

Seems like that'll be more readable